ecs-agent/tcs/client/client_test.go

```diff
@@ -132,7 +125,7 @@ func (*mockStatsEngine) SetPublishServiceConnectTickerInterval(counter int32) {
 }
 
 func (*mockStatsEngine) GetPublishMetricsTicker() *time.Ticker {
-	return time.NewTicker(config.DefaultContainerMetricsPublishInterval)
+	return time.NewTicker(DefaultContainerMetricsPublishInterval)
```

Comment:
for this and other uses of `DefaultContainerMetricsPublishInterval` below, is this now defined locally in the websocket?  Is this something that should still be centrally configurable?

Reply:
This is now defined in TCS Client, and I believe this could be either passed from outside (from each agent config) or defined locally in TCS Client. However, since both Fargate and us are using the same TCS client and same contract (20s interval per metrics publish), plus I don't see in the near future that we need to configure this value differently for Fargate and us, I moved it into the ecs-agent module.